content/en/docs/hertz/getting-started/_index.md

```diff
@@ -93,7 +93,8 @@ First, you need to install the command tool hz which is used in this chapter:
 1. Confirm the `GOPATH` environment has been defined correctly (For example `export GOPATH=~/go`)
    and the `$GOPATH/bin` has been added to `PATH` environment (For example `export PATH=$GOPATH/bin:$PATH`);
    Attention, do not set `GOPATH` to a directory that the current user does not have read/write access to.
-2. Install hz: `go install github.com/cloudwego/hertz/cmd/hz@latest`.
+2. You may need to set up a domestic proxy: `go env -w GOPROXY=https://goproxy.cn`.
```

Comment:
For English speakers, they don't need a domestic proxy

Reply:
I have deleted the English part, thanks for the suggestion！